content/definition.md

```diff
@@ -8,7 +8,7 @@ Software is considered Ethical Source if it meets the following criteria:
 1. It is distributed with source code and can be freely used or combined with other software.
 1. It is developed or maintained in public view and welcoming of public contributions.
 1. Its community of contributors is governed by a code of conduct that is consistently and fairly enforced.
-1. It is distributed under a license that prohibits the software from being used any individual or organization which directly or indirectly facilitates, encourages, manipulates, coerces, or forces people to engage in behaviors that would violate any person’s human rights including, but not limited to, those enumerated in the Universal Declaration of Human Rights.
+1. It is distributed under a license that prohibits the software from being used any individual or organization which directly or indirectly facilitates, encourages, manipulates, coerces, or forces people to engage in behaviors that are in opposition to a specified framework of social good.
```

Comment:
Oh interesting - so the idea is that the author specifies the framework? That makes more sense than trying to pick something "universal".

Reply:
Yeah, that's the idea! This opens the way for licenses like Atmosphere, 966.ICU, and similar to fit under the umbrella.